R/zzz.R

```diff
@@ -0,0 +1,3 @@
+.onLoad <- function(...) {
```

Comment:
```suggestion
.onLoad <- function(libname, pkgname) {
```

I know we don't the args but it feels like something CRAN might decide they don't like one day